applications/OptimizationApplication/python_scripts/optimization_info.py

```diff
@@ -0,0 +1,145 @@
+import KratosMultiphysics as Kratos
```

Comment:
Thanks a lot for introducing this class. This would be hell useful. Hoeve none of its functionality is used although this PR is about this class. Please consult my general comments in the description of the PR.
We must use its funcs at the base optimization algorithm class

Reply:
addressed in my general answer